src/scripts/towns/TownList.ts

```diff
@@ -1955,6 +1955,18 @@ const FortreeRanger = new NPC('Pokémon Ranger Catherine', [
     'Please recycle your used Dowsing Machines.',
 ], {image: 'assets/images/npcs/Pokemon Ranger (female).png'});
 
+const Steven1 = new NPC('Steven', [
+    'I have been investigating the behavior of Kecleon, the Color Swap Pokémon.',
+    'There are a few hidden near the city. If you can find them all, I will give you a wonderful prize!'],
+{requirement: new TemporaryBattleRequirement('Kecleon3', 1, GameConstants.AchievementOption.less),
+});
+
+const Steven2 = new NPC('Steven', [
+    'Behold! A picture of a camouflaged Kecleon! That is your reward.',
+], {image: 'assets/images/npcs/other/KecleonDrawing.png',
```

Comment:
We can't expect players to have read the first NPC, so this might need some more context.
Other than that, i have done some code cleanup. The rest looks good

Reply:
Changed text to give more context if the player misses Steven1